test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -225,4 +225,45 @@ Describe "Get-Help should find help info within help files" -Tags @('CI', 'Requi
             Remove-Item $helpFilePath -Force -ErrorAction SilentlyContinue
         }
     }
+
+    # There is a bug specific to Travis CI that hangs the test if "get-help" is used to search pattern string. This doesn't repro locally. 
+    # This occurs even if Unix system just returns "Directory.GetFiles(path, pattern);" as the windows' code does.
+    # Since there's currently no way to get the vm from Tranvis CI and the test PASSES locally on both Ubuntu and MacOS, excluding pattern test under Unix system.
+
+    $testcases = @(
+                    @{command = "Get-Help about_testCas?1"; testname = "test ? pattern"; result = "about_test1"}
+                    @{command = "Get-Help about_testCase.?"; testname = "test ? pattern with dot"; result = "about_test2"}
+                    @{command = "(Get-Help about_testCase*).Count"; testname = "test * pattern"; result = "2"}
+                    @{command = "Get-Help about_testCas?.2*"; testname = "test ?, * pattern with dot"; result = "about_test2"}
+                 )
+
+    It "Get-Help should find pattern help files - <testname>" -TestCases $testcases -skip: (-not $IsWindows){
```

Comment:
unless the testcases are used in other tests, I would prefer to have these inline to the `It` statement as it makes it easier to associate the two